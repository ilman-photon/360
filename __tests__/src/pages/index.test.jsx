import { render, screen } from "@testing-library/react";
import Home from "../../../src/pages/index";
import "@testing-library/jest-dom";

describe("Home", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
