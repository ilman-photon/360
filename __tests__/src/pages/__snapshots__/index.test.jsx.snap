// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div
    class="authLayout"
  >
    <div
      class="authContainer"
    >
      <section
        class="authComponentContainer"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root custom-button custom-button--large  css-13vkird-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Logout
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </section>
      <section
        class="authImageContainer"
      >
        <img
          class="imageBanner"
          src="https://c4.wallpaperflare.com/wallpaper/930/115/679/panda-4k-high-quality-hd-wallpaper-preview.jpg"
        />
      </section>
    </div>
  </div>
</div>
`;
